net/Cargo.toml

```diff
@@ -23,8 +23,13 @@ move-binary-format = { git = "https://github.com/pontem-network/diem.git", branc
 [dependencies.substrate-api-client]
 git = "https://github.com/scs/substrate-api-client.git"
 rev = "62afbac0be1928681d8c70565fa0cc39a92cf4c8"
-[dependencies.keyring]
+
+[dependencies.sp-keyring]
 version = '2.0.0'
 package = "sp-keyring"
+
 [dependencies.sp-core]
 version = '2.0.0'
+package = "sp-core"
```

Comment:
Using local dep-crate name `[dependencies.THIS]` is reasonable if it differs from the real name only, then you specifies as `package = "sp-keyring"`.

Reply:
@fzzr- Thanks. Fixed it